apps/service-catalog/components/basic-service-form/validation-schema.tsx

```diff
@@ -22,9 +22,13 @@ export const validationSchema = Yup.object().shape({
       }),
       email: Yup.string().email(localization.validation.invalidEmail),
       telephone: Yup.string().matches(telephoneNumberRegex, localization.validation.invalidTlf),
-      contactPage: Yup.string().matches(urlRegex, localization.validation.invalidUrl),
+      contactPage: Yup.string()
+        .url(localization.validation.invalidUrl)
+        .matches(httpsRegex, localization.validation.invalidUrl),
```

Comment:
Siden protokoll valideres seperat så kan du jo også legge til seperate feilmeldinger:
```suggestion
        .url(localization.validation.invalidUrl)
        .matches(httpsRegex, localization.validation.invalidProtocol),
```

Reply:
Fikset ✅